routers/repo/issue_label_test.go

```diff
@@ -119,6 +120,7 @@ func TestDeleteLabel(t *testing.T) {
 }
 
 func TestUpdateIssueLabel_Clear(t *testing.T) {
+	setting.Service.EnableIssues = true
```

Comment:
Since default value is true, this is unnecessary.

Reply:
Actually the tests fail, if this line is removed. (See https://drone.gitea.io/go-gitea/gitea/10658/1/10)